feat(react): react devtool (#236)
codesandbox/sandpack: website/docs/docs/advanced-usage/components.md

  </SandpackLayout>
</SandpackProvider>

## ReactDevTools

> Sandpack also provides a component to add a plugin for React debugging, which allows you to inspect the React component hierarchies in the iframe. This is useful for `props` debugging and understanding the component tree.

Reviewer: ```suggestion
Sandpack also provides a component that adds a plugin for React debugging, allowing you to inspect the React component hierarchies in the iframe. This is useful for `props` debugging and understanding the component tree.
```

Reviewer: (I'm not 100% sure if "instance" is the right word there at the end??)

  </SandpackLayout>
</SandpackProvider>

## ReactDevTools

Sandpack also provides a component that adds React DevTools, allowing you to inspect the React component hierarchies in the iframe. This is useful for `props` debugging and understanding the component tree. Our `SandpackReactDevTools` component has the same functionality as the React DevTools browser extensions, but it only shows what is in your Sandpack instance.

<!-- prettier-ignore -->
<div className="nestedSandpack">
  <Sandpack
    theme="sandpack-dark"
    customSetup={{
      dependencies: { "@codesandbox/sandpack-react": "latest" },
    }}
    files={{
      "/App.js": `import {
  SandpackProvider,
  SandpackLayout,
  SandpackPreview,
  SandpackReactDevTools,
} from "@codesandbox/sandpack-react";
import "@codesandbox/sandpack-react/dist/index.css";\n
export default function CustomSandpack() {
  return (
    <SandpackProvider template="react">
      <SandpackLayout>
        <SandpackPreview />
        <SandpackReactDevTools style={{ width: "50%" }} />
      </SandpackLayout>
    </SandpackProvider>
  )

> }`

Reviewer 2: Looks like you forgot the "```" ticks for formatting here.
